cpp/log/cluster_state_controller-inl.h

```diff
@@ -446,11 +449,14 @@ void ClusterStateController<Logged>::DetermineElectionParticipation(
 template <class Logged>
 void ClusterStateController<Logged>::ClusterServingSTHUpdater() {
   while (true) {
+    LOG(INFO) << "ClusterServingSTHUpdater going again.";
     std::unique_lock<std::mutex> lock(mutex_);
     update_required_cv_.wait(lock, [this]() {
       return update_required_ || exiting_;
     });
+    LOG(INFO) << "ClusterServingSTHUpdater got ping.";
     if (exiting_) {
+      LOG(INFO) << "ClusterServingSTHUpdater thread returning.";
```

Comment:
Not sure if those should be ```VLOG``` or not, but this is fine for now, we can change it later.